nixos/modules/services/misc/plex.nix

```diff
@@ -91,7 +91,7 @@ in
         # Copy the database skeleton files to /var/lib/plex/.skeleton
         # See the the Nix expression for Plex's package for more information on
         # why this is done.
-        test -d "${cfg.dataDir}/.skeleton" || mkdir "${cfg.dataDir}/.skeleton"
+        test -d "${cfg.dataDir}/.skeleton" || (mkdir "${cfg.dataDir}/.skeleton" && chown ${cfg.user}:${cfg.group} "${cfg.dataDir}/.skeleton")
```

Comment:
I think the 'install' version will reset owner/group on the ".skeleton" directory should they be changed after creation.  Similar to running the chown part of the command unconditionally.

I don't remember the details of how this is used but I seem to recall not changing permissions of existing directory might be important? Dunno.